doc/debugging-an-app.md

```diff
@@ -6,19 +6,24 @@ docpage = true
 +++
 
 This section assumes you've followed the steps in the [Getting
-Started](/getting-started) guide and have Conduit and the demo application
+Started](/getting-started) guide and have Linkerd and the demo application
 running in some flavor of Kubernetes cluster.
 
-## Using Conduit to debug a failing service 💻🔥
-Now that we have Conduit and the demo application [up and
-running](/getting-started), let's use Conduit to diagnose issues.
+## Using Linkerd to debug a failing service 💻🔥
 
-First, let's use the `conduit stat` command to get an overview of deployment
+Now that we have Linkerd and the demo application [up and
+running](/getting-started), let's use Linkerd to diagnose issues.
+
+First, let's use the `linkerd stat` command to get an overview of deployment
 health:
-#### `conduit -n emojivoto stat deploy`
```

Comment:
Switching these h4s to code blocks is OK, but it means that the style for input commands on this page no longer matches the style used on the Getting Started page, since you're not making an equivalent change in that document. My preference is to defer style changes like this one to a separate branch.

Reply:
Sounds great to me.